atd-vze/src/views/Users/Users.js

```diff
@@ -66,30 +113,52 @@ const Users = () => {
 
   const [userList, setUserList] = useState(null);
   const [totalUsers, setTotalUsers] = useState(null);
-  const [page, setPage] = useState(0);
+  const [currentPage, setCurrentPage] = useState(0);
+  const [allUsers, setAllUsers] = useState([]);
+  const [isFetchingAllUsers, setIsFetchingAllUsers] = useState(false);
+  const [copyUserEmailsClicked, setCopyUserEmailsClicked] = useState(false);
   const perPage = 50;
 
+  // Sets the user list for the current page and the total number of users for all pages
   useEffect(() => {
-    const endpoint = `${process.env.REACT_APP_CR3_API_DOMAIN}/user/list_users?page=${page}&per_page=${perPage}`;
-    axios
-      .get(endpoint, {
-        headers: {
-          Authorization: `Bearer ${token}`,
-        },
-      })
-      .then(res => {
-        setUserList(res.data.users);
-        setTotalUsers(res.data.total);
-      });
-  }, [token, page, perPage]);
+    getCurrentPageUsers(currentPage, perPage, token).then(res => {
+      setUserList(res.data.users);
+      setTotalUsers(res.data.total);
+    });
+    // eslint-disable-next-line
```

Comment:
Neither here nor there, but I really like development experience of having a unforgiving linter and then disabling it on a per-line basis as needed. This is great to see in here.